test/helpers/db.js

```diff
@@ -9,18 +9,10 @@ const Notification = require('../../src/models/db/notification').Notification
 const Fulfillment = require('../../src/models/db/conditionFulfillment').ConditionFulfillment
 
 exports.init = function * () {
+  yield knex.migrate.rollback(knexConfig).then()
```

Comment:
Just running `npm run test-oracle` on the commit titled "Update based on feedback".

`alter table "transfers" add primary key ("id") - ORA-02260: table can have only one primary key`

 I'm not sure what the difference is between your setup and mine.